db/seeds.rb

```diff
@@ -79,15 +79,54 @@ def position(type)
   surname = player['web_name']
   full_name = "#{player['first_name']} #{player['second_name']}"
   position = position(player['element_type'])
+
+   # Get specific player url
+   url = 'https://fantasy.premierleague.com/drf/element-summary/' << player['id'].to_s
+   response = HTTParty.get(url)
+ 
+    # Get specific player url
+    url = 'https://fantasy.premierleague.com/drf/element-summary/' << player['id'].to_s
+    response = HTTParty.get(url)
+  
+    # Gives hisory of the player
+    history_raw = JSON.parse(response.body)['history']
+ 
+  
+    # Gives the round number as the key
+    round_as_key = history_raw.collect { |p| {p['round'] => p['total_points'] }}
```

Comment:
use a different variable than `p`. `p` is another way to print out

Reply:
Updated